Makefile

```diff
@@ -37,17 +41,15 @@ crowdin-download: test-crowdin
 ###
 
 BUNDLE_EXISTS := $(shell command -v bundle 2> /dev/null)
-JEKYLL_EXISTS := $(shell command -v jekyll 2> /dev/null)
 CROWDIN_EXISTS := $(shell command -v crowdin-cli 2> /dev/null)
+YARN_EXISTS := $(shell command -v yarn 2> /dev/null)
 
-test-bundler:
+test-builddeps:
 ifndef BUNDLE_EXISTS
 	$(error bundler is not installed. Run `gem install bundler`)
 endif
-
-test-jekyll:
-ifndef JEKYLL_EXISTS
-	$(error Jekyll is not installed. Run `make install`)
```

Comment:
Why remove this? The site still depends on Jekyll so the script should probably check that it exists.